test/functional/JIT_Test/testng.xml

```diff
@@ -438,6 +438,11 @@
 	 <classes>
 	   <class name="jit.test.tr.SIMDOpts.SIMDOptTest" />
 	 </classes>
+  </test>
+  <test name="BNDCHKSimplifyTest">
+	 <classes>
+	   <class name="jit.test.tr.SIMDOpts.SIMDOptTest" />
```

Comment:
is that right? it doesn't look right it should be the class you added no?

Reply:
@andrewcraik my bad, I had this one fixed, but somehow missed during the git push. I will update it. Thanks.